src/StackExchange.Redis/Interfaces/IDatabase.cs

```diff
@@ -472,6 +472,18 @@ public interface IDatabase : IRedis, IDatabaseAsync
         /// <returns>The endpoint serving the key.</returns>
         EndPoint IdentifyEndpoint(RedisKey key = default(RedisKey), CommandFlags flags = CommandFlags.None);
 
+        /// <summary>
+        /// Copy key from the selected key to the specified destination key.
+        /// </summary>
+        /// <param name="source">Copy the value stored at the source.</param>
+        /// <param name="destination">Copy the value to destination in the logical database used by the connection.</param>
+        /// <param name="destinationDatabase">Specifying an alternative logical database index for the destination key.</param>
+        /// <param name="replace">Removes the destination key before copying the value to it.</param>
+        /// <param name="flags">The flags to use for this operation.</param>
+        /// <returns><see langword="true"/> if key was copied. <see langword="false"/> if key was not copied.</returns>
+        /// <remarks>https://redis.io/commands/copy</remarks>
+        bool KeyCopy(RedisKey source, RedisKey destination, int destinationDatabase = -1, bool replace = false, CommandFlags flags = CommandFlags.None);
```

Comment:
Recommend we call these `sourceKey` and `destinationKey` to be super clear. The documentation above could be between 2 servers for example (confused with migrate). Let's please explicitly say `key` there too (happy to tweaks docs and push if you want!)